api/processors/group.py

```diff
@@ -83,6 +127,29 @@ def _set_students_status_start(group: Group, timestamp: datetime.datetime) -> No
             status_since=timestamp,
         )
 
+    @staticmethod
+    def _set_students_status_abort(group: Group, timestamp: datetime.datetime) -> None:
+        group.students.update(
+            status=StudentStatus.AWAITING_OFFER,
+            status_since=timestamp,
+        )
+
+    @staticmethod
+    def _mark_group_linked_objects_as_former(group: Group) -> None:
+        teachers_current, students_current, coordinators_current = (
+            group.teachers,
+            group.students,
+            group.coordinators,
+        )
+
+        group.teachers.set([])
```

Comment:
Let's use .clear() here and below

https://docs.djangoproject.com/en/4.2/ref/models/relations/#django.db.models.fields.related.RelatedManager.clear